src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final int MAX_VALUE = 5;
+    private static final Random random = new Random();
+    private final ColorSupplier colorSupplier;
+
+    public FigureSupplier(ColorSupplier colorSupplier) {
+        this.colorSupplier = colorSupplier;
+    }
+
+    public static Figure getDefaultFigure() {
+        return new Circle("White", 10);
+    }
```

Comment:
Don't use static methods in your solution. Instead of making `getDefaultFigure` static, create an instance method that can be called on an object of `FigureSupplier`.